core/app/models/spree/tax/order_adjuster.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module Tax
+    class OrderAdjuster
+      attr_reader :order
+
+      include TaxHelpers
+
+      def initialize(order)
+        @order = order
+      end
+
+      def adjust!
+        return unless order_tax_zone
+
+        (order.line_items + order.shipments).each do |item|
```

Comment:
Is this a concept we should have on the order model? Some sort of method that returns all things that can be adjusted?